lib/req_athena/result.ex

```diff
@@ -10,23 +10,32 @@ defmodule ReqAthena.Result do
     * `statement_name` - The statement name from executed query;
     * `query_execution_id` - The id from executed query;
     * `output_location` - The S3 url location where the result was output.
+    * `metadata` - The columnInfo from https://docs.aws.amazon.com/athena/latest/APIReference/API_GetQueryResults.html
   """
 
   @type t :: %__MODULE__{
           columns: [String.t()],
           rows: [[term()]],
           statement_name: binary(),
           query_execution_id: binary(),
-          output_location: binary()
+          output_location: binary(),
+          metadata: [term()]
```

Comment:
```suggestion
          metadata: %{column_infos: [term()]}
```

Reply:
I changed it back because the type is for the ReqAthena.Result not the Table.Reader

Reply:
Yeah, but I think if we use a generic term like metadata (which makes sense because we are just passing somewhat unparsed information), then it's better to be a map, so we can always put more things there in the future :)